packages/block-library/src/gallery/edit.js

```diff
@@ -466,6 +470,22 @@ export default compose( [
 			resizedImages,
 		};
 	} ),
+	withDispatch( ( dispatch, { clientId }, { select } ) => {
+		return {
+			wasBlockJustInserted() {
+				const { clearLastBlockInserted } = dispatch( 'core/editor' );
+				const { wasBlockJustInserted } = select( 'core/editor' );
```

Comment:
The block-library should ideally not be relying on the  "editor" package because that package is WP specific and the blocks should ideally be generic (this rule is not always 100% respected).


In this situation, it seems, these selectors  are better placed in the block-editor package.

Reply:
It also doesn't seem like these selectors exist at all in web.

Reply:
Understood! Thanks!